src/pages/resultsView/ResultsViewPageStore.ts

```diff
@@ -273,8 +273,10 @@ export enum SampleListCategoryType {
 }
 
 export enum GeneticEntityType {
-    'GENE' = 'gene',
-    'GENESET' = 'geneset',
+    'GENE' = 'GENE',
```

Comment:
@dippindots you worked on geneset stuff right? could you take a look at these changes I made relating to GeneticEntityType and make sure I haven't done anything weird?

Reply:
@adamabeshouse This looks good to me! It's in sync with the backend.